.changeset/nervous-plants-carry.md

```diff
@@ -0,0 +1,5 @@
+---
```

Comment:
If you rename this `techdocs-nervous-plants-carry.md`, I think we can avoid core team review requirement.

Reply:
Updated